single_upload.py

```diff
@@ -81,42 +101,40 @@
 print("Enter the tag for the media:")
 mediaTag = input()
 
-url = f"https://api.jwplayer.com/v2/sites/{siteId}/media"
-payload = {
-    "upload": {
-        "method": "direct"
+response = requests.post(
```

Comment:
I like to see the parameters to a HTTP Call in the call itself, but this is also just a preference I've picked up.